csfieldguide/templates/interactives/packet-attack.html

```diff
@@ -0,0 +1,40 @@
+{% extends interactive_mode_template %}
+
+{% load i18n %}
+{% load static %}
+
+{% block html %}
+  <div class="my-3 container-fluid">
```

Comment:
Are we able to center this?

Reply:
No, I've tried a lot of things but the canvas decides it must be left-aligned